source/MaterialXTest/ShaderValid.cpp

```diff
@@ -351,33 +352,18 @@ TEST_CASE("GLSL Validation from HwShader", "[shadervalid]")
         const std::string lightDoc = " \
         <?xml version=\"1.0\"?> \
         <materialx version=\"1.35\" require=\"\"> \
-          <nodegraph name=\"lighting1\" type=\"\" xpos=\"10.9034\" ypos=\"15.42\" adskDisplayMode=\"1\"> \
```

Comment:
Simplified the test shader to a single white diffuse BSDF, to see the lighting effects better.